h2o-r/h2o-package/R/adaboost.R

```diff
@@ -28,6 +28,7 @@
 #' @param weak_learner Choose a weak learner type. Defaults to AUTO, which means DRF. Must be one of: "AUTO", "DRF", "GLM", "GBM",
 #'        "DEEP_LEARNING". Defaults to AUTO.
 #' @param learn_rate Learning rate (from 0.0 to 1.0) Defaults to 0.5.
+#' @param weak_learner_params Customized parameters for the weak_learner algorithm.
```

Comment:
Can you add example input here? Can you test if the input is correct?

Reply:
Yes, I will add it.

R test is here: h2o-r/tests/testdir_algos/adaboost/runit_adaboost_weak_learner_smoke.R

Reply:
Done